### DISCUSSION TITLE
监听127.0.0.1的socks5服务器时，无法访问谷歌

### DISCUSSION THREAD

**Discussion by someonewating** [v2rayA/v2rayA#1460] [Q&A] [answered]
Hi community,

我将v2raya运行于Ubuntu中并开启了ipv4 forwarding。这台电脑同时还运行xray，并在本地10808端口监听socks5流量。

现在我在设置v2raya时设置server为127.0.0.1:10808的服务器，并通过 `curl -v https://google.com` 确认10808端口可以正常访问谷歌。因为我的xray已经在进行分流了，因此我的v2raya配置如下：
![image](https://github.com/user-attachments/assets/fb97a029-8e76-45f9-876e-6e9dd0e9e52b)
这里的dns指向了我自己的adguard home，这个dns服务器已经正确配置过了，不会受到dns污染。

现在我发现如果我使用本地socks5服务器，则使用v2raya进行透明代理时无法访问google，但国内网站可以访问。

如果v2raya监听在同一内网内的另一个主机上的xray的socks5服务器，则可以正常访问谷歌。两台主机上的xray使用了同一份配置。

目前我观察到如果v2raya监听`127.0.0.1`，则xray后台会显示大量对我xray服务器域名的请求。而监听内网内其他设备`192.168.x.x`上的xray socks5端口则不会，且x2raya工作正常。

请问：
1. 这个是预期的行为吗？
2. 如果不是，是我哪里配置错了吗？

提前拜谢！

**Comment by eronez** [comment]
just search in google or else
"socks name resolve"

**Reply by someonewating** [reply]
> just search in google or else "socks name resolve"

Thanks mate, will try it.

**Reply by someonewating** [reply]
I will not be able to test this shortly. I will update here if I have any results.
Marking this post as done.

**Comment by someonewating** [comment, marked as answer]
使用防火墙将入站流量redirect到xray的dokedemo-door解决了。
